database.sql

```diff
@@ -0,0 +1,23 @@
+-- Delete Tables
+DROP TABLE "projects-tech-junction";
+DROP TABLE "technologies";
+DROP TABLE "projects";
+
+-- Database Setup
+CREATE TABLE "technologies" (
+	"id" SERIAL PRIMARY KEY,
+	"name" VARCHAR (255)
+);
+
+CREATE TABLE "projects" (
+	"id" SERIAL PRIMARY KEY,
+	"name" VARCHAR (255),
+	"duration" VARCHAR (255),
+	"description" TEXT
+);
+
+CREATE TABLE "projects-tech-junction" (
+	"id" SERIAL PRIMARY KEY,
+	"project_id" INTEGER REFERENCES "projects",
+	"technology_id" INTEGER REFERENCES "technologies"
+);
```

Comment:
Just a general note, and probably a personal preference, but for a site that is essentially just a portfolio, I don't think adding a DB is generally a great idea.  Just because you know how to add a DB doesn't mean you need one.  Since there is no actual user interaction where data is being added or deleted, you would be kind of needlessly complicating something that I feel should be pretty simple.

Reply:
Your projects are going to show that you know how to add and utilize a db.  If you want to do more work with DBs, start a new side project where data is going to be moving around more.  But, that's all just my opinion.